loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlow.java

```diff
@@ -96,7 +96,7 @@ public String getVersion() {
      */
     public static Runner find(String name) {
         return new Runner(PlatformConfigNamedProvider.Finder
-                .findBackwardsCompatible(name, "load-flow", LoadFlowProvider.class,
+                .find(name, "load-flow", LoadFlowProvider.class,
```

Comment:
`findBackwardsCompatible` method is declared as deprecated but indeed it might be necessary to ensure compatibility with old load flow configurations, I can go back to this method.